### ISSUE TITLE
Overwrite existing table of contents in README

### ISSUE THREAD

**Issue by kagarlickij** [alexander-lee/markdown-github-bear-toc#8]
I frequently update README.md and want to use `markdown-toc` for table of contents generation

However each time when I run `markdown-toc -h 3 -t github README.md` is fails with `[WARNING]: README.md already has a Table of Contents, Ignoring...`

I there a way to run `markdown-toc` in mode (force?) when it overwrites existing table of contents in the same file?

**Comment by alexander-lee** [comment]
Yeah this is an apparent fault :(, a solution here would be to create a function that tries to identify the start and end of a Table of Contents and overwrites it. Feel free to write up a PR!
